### PR TITLE
137 role permissions

### PR DESCRIPTION
**Pull request by Chetan0178**

### CODE REVIEW

File: apps/users/models.py [hunk 1]
```diff
@@ -129,42 +99,37 @@ def profile_picture(instance, filename):
...
 #====
 class User(AbstractBaseUser):
-    GENDER_CHOICES = [
-        ('Male', 'Male'),
-        ('Female', 'Female'),
-        ('Other', 'Other'),
-        ('Prefer Not to Say', 'Prefer Not to Say')
-    ]
+    GENDER_CHOICES = [('Male', 'Male'),('Female', 'Female'),('Other', 'Other'),('Prefer Not to Say', 'Prefer Not to Say')]
+    TITLE_CHOICES = [('Mr', 'Mr'),('Miss', 'Miss'),('Mrs', 'Mrs'),('Ms', 'Ms'),('Mx', 'Mx')]
```
Comment (QodeFarm) [inline apps/users/models.py:103]: Use only Mr. and Ms. and default is None

File: apps/users/url.py [hunk 2]
```diff
@@ -5,20 +5,21 @@
...
 router.register(r'userdata', GetUserDataViewSet, basename='userdata')
-router.register(r'permissions', PermissionsViewSet, basename='permissions')
 router.register(r'module_sections', ModuleSectionsViewSet, basename='module_sections')
-router.register(r'role_permissions', RolePermissionsViewSet, basename='role_permissions')
 
 router.register(r'user_time_restrictions', UserTimeRestrictionsViewSet, basename='user_time_restrictions')
 router.register(r'user_allowed_weekday', UserAllowedWeekdaysViewSet, basename='user_allowed_weekday')
-router.register(r'user_permissions', UserPermissionsViewSet, basename='user_permissions')
+
+router.register(r'user_roles', UserRoleViewSet, basename='user_role')
```
Comment (QodeFarm) [inline apps/users/url.py:21]: change the basename='user_roles'

File: database/cnl.sql [hunk 3]
```diff
@@ -234,62 +222,34 @@ CREATE TABLE IF NOT EXISTS users (
     last_login TIMESTAMP,
     date_of_birth DATE,
     gender ENUM('Male', 'Female', 'Other', 'Prefer Not to Say'),
+    title ENUM('Mr', 'Miss', 'Mrs', 'Ms', 'Mx'),
```
Comment (QodeFarm) [inline database/cnl.sql:225]: Have only Mr. and Ms.

File: apps/users/models.py (hunk 1 above)
Comment (Chetan0178) [reply apps/users/models.py]: Done

File: apps/users/url.py (hunk 2 above)
Comment (Chetan0178) [reply apps/users/url.py]: done

File: database/cnl.sql (hunk 3 above)
Comment (Chetan0178) [reply database/cnl.sql]: Done

**Pull request merged**